catalog/forms.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+
+from django import forms
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+
+
+
+class RenewBookForm(forms.Form):
+    renewal_date = forms.DateField(help_text="Enter a date between now and 4 weeks (default 3).")
+
+    def clean_renewal_date(self):
```

Comment:
Doc string and type hint would be useful here.

Reply:
The docstring and type hint are added now